pkg/apis/serving/v1/revision_validation_test.go

```diff
@@ -284,10 +293,11 @@ func TestContainerConcurrencyValidation(t *testing.T) {
 
 func TestRevisionSpecValidation(t *testing.T) {
 	tests := []struct {
-		name string
-		rs   *RevisionSpec
-		wc   func(context.Context) context.Context
-		want *apis.FieldError
+		name    string
+		rs      *RevisionSpec
+		wc      func(context.Context) context.Context
+		cfgOpts []configOption
```

Comment:
I think we can just put the modified config into `test.wc` directly instead of introducing a new field here. Same in the v1a1 test.